adapter_pipelines/cellranger/adapter.wdl

```diff
@@ -190,39 +165,29 @@ workflow Adapter10xCount {
       pipeline_tools_version = pipeline_tools_version
   }
 
-  # Cellranger code in 10x count wdl requires files to be named a certain way.
-  # To accommodate that, RenameFastqFiles copies the blue box files into the
-  # cromwell execution bucket but with the names cellranger expects.
-  # Putting this in its own task lets us take advantage of automatic localizing
-  # and delocalizing by Cromwell/JES to actually read and write stuff in buckets.
-  # TODO: Replace scatter with a for-loop inside of the task to avoid creating a
-  # VM for each set of files that needs to be renamed
-  scatter(i in range(length(GetInputs.lanes))) {
-    call RenameFastqFiles as prep {
-      input:
-        r1 = GetInputs.r1_fastq[i],
-        r2 = GetInputs.r2_fastq[i],
-        i1 = GetInputs.i1_fastq[i],
-        sample_id = GetInputs.sample_id,
-        lane = GetInputs.lanes[i],
-        pipeline_tools_version = pipeline_tools_version
-      }
-    }
+  # Rename the fastq files to the format required by CellRanger:
```

Comment:
If I understand it correctly, cellranger doesn't support scatter tasks anyway, so we could remove the `lane` parameter here right?

Reply:
We still need to rename the fastq files, but now this is passing in an array of files instead of doing a scatter